src/services/formatting/smartIndenter.ts

```diff
@@ -277,6 +283,15 @@ module ts.formatting {
             return undefined;
         }
 
+        function getActualIndentationForBlockFormItem(node: Node, sourceFile: SourceFile, options: EditorOptions): number {

```

Comment:
Can you document what a block-form item is? Alternatively, we could think of a different name for this idea.